functions/zoom-uploader.py

```diff
@@ -288,19 +274,117 @@ def ingest(self):
             ("spatial", (None, "Zoom"))
         ]
 
-        for s3_filename in video_files:
-            url = self._generate_presigned_url(s3_filename)
-            params.extend([
-                ("flavor", (None, escape(ZOOM_OPENCAST_FLAVOR))),
-                ("mediaUri", (None, url))
-            ])
+        fpg = FileParamGenerator()
```

Comment:
Should be
`FileParamGenerator(self.s3_filenames)`
and 
fpg.generate(self.s3_filenames) should just be `fpg.generate()`